iOS/WAStickersThirdParty/Interoperability.swift

```diff
@@ -17,6 +17,10 @@ struct Interoperability {
     static var iOSAppStoreLink: String?
     static var AndroidStoreLink: String?
 
+    static func canSend() -> Bool{
```

Comment:
Can you replace the code in line 46 to use this new function?

Reply:
Good point! Done, see additional commit.